sklearn/neighbors/binary_tree.pxi

```diff
@@ -654,6 +654,56 @@ cdef class NeighborsHeap:
 
         return 0
 
+    cdef int cpush(self, ITYPE_t row, DTYPE_t val,
```

Comment:
Surely `push` can just call `cpush` (if `push` really needs to be Python-visible.